src/Actions/directActions.ts

```diff
@@ -409,15 +409,22 @@ export class CombineAction extends Action {
      */
     public children: Action[];
 
+    /**
+     * defines if the children actions conditions should be check before execution
+     */
+    public enableChildrenConditions: boolean;
+
     /**
      * Instantiate the action
      * @param triggerOptions defines the trigger options
      * @param children defines the list of aggregated animations to run
      * @param condition defines the trigger related conditions
+     * @param enableChildrenConditions defines if the children actions conditions should be check before execution
      */
-    constructor(triggerOptions: any, children: Action[], condition?: Condition) {
+    constructor(triggerOptions: any, children: Action[], condition?: Condition, enableChildrenConditions = false) {

```

Comment:
feels like this should be on per default?

Reply:
so basically back compat :-) but also there are 2 cases:
1. we consider that combined action is the one having the condition so we disregard children ones in this case which is the way we were seeing it before the forum post. Basically, the children should launch as long as the parent is ok. (handy to combine existing actions)
2. We want to also honor the children conditions as stated in the forum post

I am not sure of the ratio of both intents so in doubt I chose back compat lol